google-cloud-contrib/google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemTest.java

```diff
@@ -70,6 +70,28 @@ public void before() {
   }
 
   @Test
+  public void checkDefaultOptions() throws IOException {
```

Comment:
Can you add a test proving that the statically-set default propagates all the way to `CloudStorageReadChannel.maxChannelReopens`?